Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -18,36 +18,36 @@ class DisplayHymnViewModel: ObservableObject {
 
     private let analytics: AnalyticsLogger
     private let backgroundQueue: DispatchQueue
-    private let favoritesStore: FavoritesStore
     private let historyStore: HistoryStore
     private let identifier: HymnIdentifier
     private let mainQueue: DispatchQueue
     private let pdfLoader: PDFLoader
     private let repository: HymnsRepository
     private let storeInHistoryStore: Bool
+    private let favoriteStore: FavoriteStore
```

Comment:
So I try to keep things in alphabetical order as much as I can, so can you put this back to where it was? (i.e. line 21)